docs/kafka/getting-started-kafka/README.adoc

```diff
@@ -283,6 +283,9 @@ image::sak-create-topic.png[Image of wizard to create a topic]
 * *Replicas*: For this release of {product-kafka}, the replicas are preconfigured. The number of partition replicas for the topic is set to `3` and the minimum number of follower replicas that must be in sync with a partition leader is set to `2`. Replicas are copies of partitions in a topic. Partition replicas are distributed over multiple brokers in the cluster to ensure topic availability if a broker fails. When a follower replica is in sync with a partition leader, the follower replica can become the new partition leader if needed.
 
 After you complete the topic setup, the new Kafka topic is listed in the topics table. You can now start producing and consuming messages to and from this topic using services that you connect to this instance.
+
+NOTE: If your topic creation is unsuccessful and you receive an error message, try to create your topic again later. This issue might be due to broker unavailability.
```

Comment:
I like the brevity of this note, @fmcdonal. However, I wonder if it might be a _little_ opaque, particularly to getting started users. Specifically, I don't think we should assume that users will know what "broker unavailability" means.

I wonder if we could consider something like this: "**NOTE**: If your topic creation is unsuccessful and you receive an error message, try to create your topic again later. This situation might occur, for example, if your selected cloud provider has a temporary availability problem that affects your Kafka instance."

We should run any revised wording (such as my suggestion above) by Keith.

Reply:
applied your suggestion

Reply:
@kstanley could you review at your convenience please? thanks